docs/concepts.rst

```diff
@@ -842,61 +842,135 @@ values by "case" instead of by entity.
     # Returns `{'black cat', 'brown cat', 'brown fox'}`.
     builder.build().get('colored_animal', 'set')
 
-Other Features
---------------
+Parallel and Cloud Execution
+----------------------------
+
+By default, Bionic computes values one at a time. Requesting an entity value
+with :meth:`Flow.get <bionic.Flow.get>` can lead to a long computation, as
+Bionic may need to compute that entity's dependencies, and their dependencies,
+and so on.
+
+Bionic can take advantage of more computing resources by using local
+multiprocessing or Google AI Platform. However, this feature is not enabled
```

Comment:
For the "However" sentence, I would just say "Depending on the structure of your dependency graph, this may speed up your computation." In particular, I don't think the serialization requirement makes sense as a reason not to enable this by default, since user functions are unlikely to be non-serializable, and non-serializable entities are already handled.